crates/rpc/src/v05/method/get_transaction_receipt.rs

```diff
@@ -141,6 +143,8 @@ pub mod types {
         pub events: Vec<Event>,
         #[serde(skip_serializing_if = "Option::is_none")]
         pub revert_reason: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        pub execution_resources: Option<ExecutionResourcesProperties>,
```

Comment:
This one is required I think. Its only optional in the pending receipt according to the spec.

Reply:
Good point, it's actually required in both.
That could have been more obvious :sweat_smile: the `required` section at the end of the object attributes in the spec is impractical.

Reply:
@Mirko-von-Leipzig `None` will result in an internal error. Is this fine?

Reply:
I'm trying to think why it can be `None` in the first place.. but I think rather just default them to zero instead.

Reply:
Is suppose if the record is absent in DB then we get a `None` and from the little I know the most logical explanation would be a bug.
I think the internal error is a better fit, but you'll know better.
Zeroes?

Reply:
I think the issue was that the gateway originally did not provide these values and are therefore missing in some legacy databases. However since we've just forced a resync such cases should no longer exist.

That said, these values are not that important -- and are not actually verifiable as they don't form part of the hash. I think a user would rather get the receipt data than always get an error for one they are interested in.

We should create a follow-up issue to investigate if we can't remove the db optional entirely now.